data/update.py

```diff
@@ -141,41 +117,40 @@ def run(options):
 
 # Upload the scan + processed data to /live/ and /archive/ locations by date.
 def upload(date):
-  live_scanned = "s3://%s/live/scan/" % (BUCKET_NAME)
-  live_cached = "s3://%s/live/cache/" % (BUCKET_NAME)
-  live_db = "s3://%s/live/db/" % (BUCKET_NAME)
+  live_parents = "s3://%s/live/parents/" % (BUCKET_NAME)
   live_subdomains = "s3://%s/live/subdomains/" % (BUCKET_NAME)
-  archive_scanned = "s3://%s/archive/%s/scan/" % (BUCKET_NAME, date)
-  archive_cached = "s3://%s/archive/%s/cache/" % (BUCKET_NAME, date)
-  archive_db = "s3://%s/archive/%s/db/" % (BUCKET_NAME, date)
+  live_db = "s3://%s/live/db/" % (BUCKET_NAME)
   archive_subdomains = "s3://%s/archive/%s/subdomains/" % (BUCKET_NAME, date)
+  archive_parents = "s3://%s/archive/%s/parents/" % (BUCKET_NAME, date)
+  archive_db = "s3://%s/archive/%s/db/" % (BUCKET_NAME, date)
 
   acl = "--acl=public-read"
 
-  shell_out(["aws", "s3", "sync", SCANNED_DATA, live_scanned, acl])
-  shell_out(["aws", "s3", "sync", CACHE_DATA, live_cached, acl])
+  shell_out(["aws", "s3", "sync", PARENTS_DATA, live_scanned, acl])
   shell_out(["aws", "s3", "sync", SUBDOMAIN_DATA, live_subdomains, acl])
   shell_out(["aws", "s3", "cp", DB_DATA, live_db, acl])
 
   # Ask S3 to do the copying, to save on time and bandwidth
   shell_out(["aws", "s3", "sync", live_scanned, archive_scanned, acl])
-  shell_out(["aws", "s3", "sync", live_cached, archive_cached, acl])
   shell_out(["aws", "s3", "sync", live_subdomains, archive_subdomains, acl])
   shell_out(["aws", "s3", "sync", live_db, archive_db, acl])
 
 
 # Use domain-scan to scan .gov domains from the set domain URL.
-# Drop the output into data/output/scan/results.
-def scan(options):
+# Drop the output into data/output/parents/results.
+def scan_parents(options):
   scanners = "--scan=%s" % SCANNERS
   analytics = "--analytics=%s" % ANALYTICS_URL
-  output = "--output=%s" % SCAN_TARGET
+  output = "--output=%s" % PARENTS_DATA
   a11y_redirects = "--a11y_redirects=%s" % A11Y_REDIRECTS
   a11y_config = "--a11y_config=%s" % A11Y_CONFIG
 
   full_command =[
     SCAN_COMMAND, DOMAINS,
-    scanners, analytics, output, a11y_config, a11y_redirects,
+    scanners,
+    analytics, a11y_config, a11y_redirects,
+    output,
+    "--sslyze-no-certs", # ugh, temporary
```

Comment:
Noting that this will change soon in a release of `domain-scan`, probably to become `--sslyze-certs=false`. Not a blocker.